src/e2e/sdk_remote_integration.spec.ts

```diff
@@ -83,6 +83,16 @@ describe("Erdstall-TS-SDK", () => {
 		return Promise.all(stages.map(stage => stage.wait()));
 	});
 
+	it("await deposit finalization", async() => {
```

Comment:
Is this a testcase? I would prefer to have it in a normal closure which is used to wait for phaseshifts, otherwise every `it` is potentially not a testcase.

Reply:
I just realise that quite a few of those testcases are no testcases but rather steps to setup the environment, or am I missing something?

Reply:
I realise this is not the scope of this PR, so this should not be done here but IMHO I think we should extract those steps into a dedicated `setup` function which then also returns a helper object with which one can wait for phaseshifts. These could then be utilised with the `before{All|Each}` hooks.

If you agree, I would appreciate an issue for this :smile:

Reply:
1. `it()` is asynchronous, and I didn't know another way of executing stuff between two tests. For now, I included it in the deposit test.
2. Stuff like subscribing clients could be seen as a test case, as subscribing isn't tested anywhere else against a real operator, AFAIK, but it's also necessary for the next tests.
3. I think I'll have to take another look at how mocha works.

Reply:
We will keep it like this for now, I created an [issue](https://github.com/perun-network/erdstall-ts-sdk/issues/79) in which this can be investigated further. As far as I am aware, it is common to create sub-testsuites (with `describe`) in which one can run the needed code in between multiple test(-suites).